apps/k8s-io/OWNERS

```diff
@@ -1,10 +1,5 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
-# deploying this is still a manual process, so restricting
-# to approvers who are comfortable doing so
-options:
-  no_parent_owners: true
-
```

Comment:
Now that we have jobs auto-deploy k8s.io changes on PR merge, this is no longer needed 🎉